docs/src/data/keymap-upgrade.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: CC-BY-NC-SA-4.0
+ */
+
+export const Codes = {
+  NUM_1: "N1",
+  NUM_2: "N2",
+  NUM_3: "N3",
+  NUM_4: "N4",
+  NUM_5: "N5",
+  NUM_6: "N6",
+  NUM_7: "N7",
+  NUM_8: "N8",
+  NUM_9: "N9",
+  NUM_0: "N0",
+  BKSP: "BSPC",
+  SPC: "SPACE",
+  EQL: "EQUAL",
+  TILD: "TILDE",
+  SCLN: "SEMI",
+  QUOT: "SQT",
+  GRAV: "GRAVE",
+  CMMA: "COMMA",
+  PRSC: "PSCRN",
+  SCLK: "SLCK",
+  PAUS: "PAUSE_BREAK",
+  PGUP: "PG_UP",
+  PGDN: "PG_DN",
+  RARW: "RIGHT",
+  LARW: "LEFT",
+  DARW: "DOWN",
+  UARW: "UP",
+  KDIV: "KP_DIVIDE",
+  KMLT: "KP_MULTIPLY",
+  KMIN: "KP_MINUS",
+  KPLS: "KP_PLUS",
+  UNDO: "K_UNDO",
+  CUT: "K_CUT",
+  COPY: "K_COPY",
+  PSTE: "K_PASTE",
+  VOLU: "K_VOL_UP",
+  VOLD: "K_VOL_DN",
+  CURU: null,
+  LPRN: "LPAR",
+  RPRN: "RPAR",
+  LCUR: "LBRC",
+  RCUR: "RBRC",
+  CRRT: "CARET",
+  PRCT: "PRCNT",
+  LABT: "LT",
+  RABT: "GT",
+  COLN: "COLON",
+  KSPC: null,
+  ATSN: "AT",
+  BANG: "EXCL",
+  LCTL: "LCTRL",
+  LSFT: "LSHFT",
+  RCTL: "RCTRL",
+  RSFT: "RSHFT",
+  M_NEXT: "C_NEXT",
+  M_PREV: "C_PREV",
+  M_STOP: "C_STOP",
+  M_EJCT: "C_EJECT",
+  M_PLAY: "C_PP",
+  M_MUTE: "C_MUTE",
+  M_VOLU: "C_VOL_UP",
+  M_VOLD: "C_VOL_DN",
+  GUI: "K_CMENU",
+};
+
+export const Behaviors = {
+  "&cp": "&kp",
+};
```

Comment:
Can we add this please?

`"&inc_dec_cp": "&inc_dec_kp"`

Thanks!

Reply:
Done.